cluster/vm-backing-store.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: VirtualMachine
+metadata:
+  name: testvm-backing-store
+spec:
+  terminationGracePeriodSeconds: 0
+  domain:
+    resources:
+      requests:
+        memory: 64M
+    devices:
+      disks:
+      - name: mydisk
+        volumeName: myvolume
+        disk:
+          dev: vda
+  volumes:
+    - name: myvolume
+      iscsi:
+        iqn: iqn.2017-01.io.kubevirt:sn.42
+        lun: 4
+        targetPortal: iscsi-demo-target.kube-system.svc.cluster.local
+      backedEphemeral:
```

Comment:
I messed up the example. The example should look like this: https://pastebin.com/UfZvXyXj

Then it reflects the intention in the code.